### PR TITLE
[cannon] Add params for IBodyOptions interface

### PR DESCRIPTION
**Pull request by ChrisAntaki**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 
        https://github.com/schteppe/cannon.js/blob/master/src/objects/Body.js#L17-L35
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/cannon/index.d.ts [hunk 1]
```diff
@@ -454,6 +454,15 @@ declare module CANNON {
         type?: number;
         linearDamping?: number;
         angularDamping?: number;
+        allowSleep?: boolean;
+        sleepSpeedLimit?: number;
+        sleepTimeLimit?: number;
+        collisionFilterGroup?: number;
+        collisionFilterMask?: number;
+        fixedRotation?: boolean;
+        linearFactor?: Vec3;
```
Comment (Gelio) [inline types/cannon/index.d.ts]: I do not see this property used in [the source code for Body](http://schteppe.github.io/cannon.js/docs/files/src_objects_Body.js.html#l12).

The same goes for `angularFactory`. Where do they come from?

Comment (ChrisAntaki) [reply types/cannon/index.d.ts]: Good catch! I've removed `linearFactor` and `angularFactor` from this PR. They are present in the master branch of canon [1], but not in the last release pushed to npm, 0.6.2.

[1] https://github.com/schteppe/cannon.js/blob/master/src/objects/Body.js#L17-L35

**Review by Gelio — APPROVED** [review]
Good job :+1: Thanks for the contribution

**Comment by ChrisAntaki** [comment]
Thanks for the review

**Pull request merged**